lib/util/config-types.ts

```diff
@@ -318,6 +318,7 @@ export interface ModesConfig {
   }
   modeButtons?: ModeButtonDefinition[]
   modeSettingDefinitions?: ModeSetting[]
+  numItineraries?: number
```

Comment:
Isn't this a config change?

Reply:
Oh, yes, it is an existing parameter that was undocumented (https://github.com/opentripplanner/otp-react-redux/pull/1235/commits/5adc6a57c6bde9d67d8f022e73bdbba2f5a6494b)